wurst/systems/trade/TradeShipList.wurst

```diff
@@ -0,0 +1,24 @@
+package TradeShipList
+import LinkedListModule
+
+public class TradeShipList
+    use LinkedListModule
+
+    int id
+    
+    function setId(int _id)
```

Comment:
I think using the same variable name without a _ and using `this` to refer to the instance variable is generally preferred.

Reply:
@jonathanmclaus I don't like using this actually. The problem with _ is that it makes the variable optional to read (instead of a warning)